docs/faq.md

```diff
@@ -241,3 +241,24 @@ grey_allOf(grey_sufficientlyVisible(),
 then all elements in the entire application will be checked for visibility, and finally one
 with a matching id will be selected. It's significantly faster to use the most targeted
 matchers first (typically `grey_accessibilityID` or `grey_accessibilityLabel`).
+
+**How do I wait for an element to appear using Swift?**
```

Comment:
This one is interesting because you wouldn't need to add explicit waits/sleeps if synchronization is setup correctly. We should mention that this is a workaround and not a best practice.